docs/Developer-Guide.md

```diff
@@ -272,6 +272,9 @@ $ script -fec 'sudo -E USE_DOCKER=true ./image_builder.sh ${ROOTFS_DIR}'
 > - If you do *not* wish to build under Docker, remove the `USE_DOCKER`
 >   variable in the previous command and ensure the `qemu-img` command is
 >   available on your system.
+>   - If `qemu-img` is not installed, you will likely see errors such as ```ERROR: File /dev/loop19p1 is not a block device```
```

Comment:
You only need a single backtick at each end of these error messages as the verbatim text is "inline".

Reply:
OK, that makes sense, thanks

Have added a 3rd commit 🤞